common/lib/xmodule/xmodule/js/src/capa/display.coffee

```diff
@@ -402,24 +405,13 @@ class @Problem
             MathJax.Hub.Queue ["Typeset", MathJax.Hub, element]
 
         `// Translators: the word Answer here refers to the answer to a problem the student must solve.`
-        @$('.show-label').text gettext('Hide Answer')
+
         @el.addClass 'showed'
+        $('html, body').animate({
```

Comment:
Isn't there already a helper method to scroll up and put focus in the right place? If not, you should make one because we will be doing this action in several different places.